lib/exe/css.php

```diff
@@ -34,98 +35,125 @@ function css_out(){
 
     if ($INPUT->str('s') == 'feed') {
         $mediatypes = array('feed');
-        $type = 'feed';
     } else {
-        $mediatypes = array('screen', 'all', 'print');
-        $type = '';
+        $mediatypes = array('screen', 'all', 'print', 'handheld');
```

Comment:
I removed 'feed' due to only being a hint for the cache. The cache - see below - now uses all css files that will be included and therefore is already a hint to the cache.